docs/3-ARCHITECTURE 2.md

```
@@ -0,0 +1,829 @@
+        output_path: str
+    ):
+        """Export model to spreadsheet output."""
+        ...
+```
+
+## Extension Development Guide
+
+### Creating a New Extension
+
+1. **Create package structure**:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)